### PR TITLE
Fix Lidar cloud point access in controllers

### LINKED ISSUE #1089: ros controller crashes with ToF/3D cam simulation
**Describe the Bug**
I would like to simulate a 3D/ToF camera. For this purpose I've set up a sim with with a Lidar device. The ToF cam has a resolution of 224x171 and FOV of 62x45deg, I've entered that in Webots. So far so good.

When I now try to use the sim with a ros controller, the controller just crashes. I suspect that this is due to the high number of layers since it doesn't crash when I reduce the layers. 

**Steps to Reproduce**
1. Create a robot with a Lidar device
2. Set the Lidar device `horizontalCameraResolution` to 224 and `numberOfLayers` to 171.
3. Set the robots controller to `ros`
4. Start the sim and enable both the lida and `enable_point_cloud`
5. The `ros` controller crashes.

**Expected behavior**
The `ros` controller works as expected or gives an appropriate error message.

**Screenshots**
none

**System**
 - Operating System: Ubuntu Bionic 18.04
 - Graphics Card:
```
 glxinfo -B
name of display: :0
display: :0  screen: 0
direct rendering: Yes
Memory info (GL_NVX_gpu_memory_info):
    Dedicated video memory: 4096 MB
    Total available memory: 4096 MB
    Currently available dedicated video memory: 2878 MB
OpenGL vendor string: NVIDIA Corporation
OpenGL renderer string: Quadro P2000/PCIe/SSE2
OpenGL core profile version string: 4.6.0 NVIDIA 410.104
OpenGL core profile shading language version string: 4.60 NVIDIA
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile

OpenGL version string: 4.6.0 NVIDIA 410.104
OpenGL shading language version string: 4.60 NVIDIA
OpenGL context flags: (none)
OpenGL profile mask: (none)

OpenGL ES profile version string: OpenGL ES 3.2 NVIDIA 410.104
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20
```

**Additional context**
Webots sim: https://wolke.roessler.systems/s/q3Map9kMnPxwYA6 `simple.wbt`

### PR DESCRIPTION
**Pull request by fabienrohrer**
Fix #1089

### CODE REVIEW

**Comment by fabienrohrer** [comment]
@DavidMansolino Could you confirm the bug in ROS is fixed with this patch?

**Comment by fabienrohrer** [comment]
Unit tests in progress: https://travis-ci.com/cyberbotics/webots/builds/136150737

**Comment by DavidMansolino** [comment]
:eyes:

**Comment by DavidMansolino** [comment]
I confirm it is working now fine with ros too.

**Comment by fabienrohrer** [comment]
👍 ✔️

File: docs/reference/changelog-r2019.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Released on ???
   - Dependency Updates
     - Upgraded to Qt 5.13.1 on Windows and macOS.
   - Bug fixes
+    - Fixed [Lidar](lidar.md) cloud point access in controllers (thanks to Alexander).
```
Comment (DavidMansolino) [inline docs/reference/changelog-r2019.md]:
```suggestion
    - Fixed [Lidar](lidar.md) point cloud access in controllers (thanks to Alexander).
```
?

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**